src/styles/blocks/card.scss

```diff
@@ -0,0 +1,59 @@
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 33px;
+  }
+
+  &__title {
+    display: flex;
+    margin: 40px 0 0;
+    width: 166px;
+
+    font-weight: $font-weight--medium;
+    font-size: 12px;
+    line-height: 18px;
+    color: $c-dark;
+  }
+
+  &__art-number {
+    display: flex;
+    align-self: flex-start;
+    margin: 4px 0 0 17px;
+
+    font-weight: $font-weight--regular;
+    font-size: 10px;
+    line-height: 14px;
+    color: $c-grey;
+  }
+
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 17px 0;
+    width: 166px;
```

Comment:
do not specify width, what if you will change the width of the card? you will need to change the width everywhere